src/ovirtapi/transform.js

```diff
@@ -1024,6 +1024,38 @@ const User = {
   toApi: undefined,
 }
 
+//
+//
+//
+const MaxNumOfVmCpus = {
+  toInternal (cpusPerArch: string): Object {
+    // For the expected cluster architecture types see:
+    //    http://ovirt.github.io/ovirt-engine-api-model/master/#types/architecture
+    const archToCount = {
```

Comment:
1. consider creating a Flow type definition
2. please add a comment showing what type of data we are expecting - the format is a non standard one
3. do we want to  support also previous format?

Reply:
>     1. consider creating a Flow type definition

Ok but it'll be generic.

> 
>     2. please add a comment showing what type of data we are expecting - the format is a non standard one
> 

Ok.

>     3. do we want to  support also previous format?

No.  If it doesn't parse, everything will still work but the max values with be stuck at 1.

Maybe it makes sense to use your pr #1384 and setup semver checking like is done in rpm spec files and keep that up to date.